packages/webapp/components/layouts/AccountLayout/Profile/VerifiedCompanyBadge/VerifiedCompanyBadgeSection.tsx

```diff
@@ -159,13 +159,20 @@ export const VerifiedCompanyBadgeSection = ({
 
   const UserHasCompanyInReview = () => {
     return (
-      <div className="mt-6 max-w-sm">
+      <div className="mt-4 flex max-w-sm flex-wrap items-center">
```

Comment:
Adjust margin-top so it would visually be similar to how it was before the button